### DISCUSSION TITLE
Paste Input variants for success, neutral, warning?

### DISCUSSION THREAD

**Discussion by WillCKyle** [twilio-labs/paste#3100] [Q&A] [answered]
Hello,

I would like to make use of the input variants that are built out in the official Twilio Paste Figma design kit, including success, neautral and warning states. I see that these are not mentioned in the official documentation for the input component. Is this on the development roadmap? Or is there a recommended way of providing design tokens conditionally to the border color, help text and icon within input?

**Comment by serifluous** [comment, marked as answer]
Hi @WillCKyle ! Can you share a screenshot (and link, if possible) of what you mean for the success, neutral, and warning states of the input in the Paste Figma kit?

We don't provide those states on our Input component, but perhaps we're referring to different things. Thanks!

**Reply by WillCKyle** [reply]
Sure! I just discovered that Twilio's Figma component library is now sunsetted and there is a new version which does not include these input variants. But I would still like to replicate this behavior, so I would love some guidance on that and what the thinking behind these originally was. Here's what I see in Figma:

![image](https://user-images.githubusercontent.com/44306769/225988848-dc9956b1-3710-40a6-b04d-bcb72e075c30.png)

**Reply by serifluous** [reply]
Got it, thanks! It seems like what may have happened is that whoever created the library in your screenshot duplicated the Twilio Figma component library into their account and added those variants themselves. If you take a look at the original library in Figma Community, it doesn't have those variants either, unfortunately: https://www.figma.com/community/file/1012844215105752327/%5BSUNSET%5D-Twilio-Paste-Components

You can check out the response here on ways you can approach styling and customization on top of our given components: https://github.com/twilio-labs/paste/discussions/1902#discussioncomment-1421995
